docs/components/popovers-tooltips.md

```diff
@@ -16,7 +16,8 @@ eleventyNavigation:
 - Popovers and Tooltips may appear at the top, right, bottom, or left of the connected object.
 - If space is limited, both will automatically adjust their orientation regardless of initial placement.
 - Popovers have a header. It’s useful when explaining things in more detail than a Tooltip.
-- Tooltips do not have a header. It’s useful only short amount of text is needed.
+- Tooltips do not have a header. It’s useful when only short amount of text is needed.
+- Consider making boxes appear on click instead of hover for accessibility (with `data-trigger="click"`).
```

Comment:
I'd like to encourage further reading for things like this instead of "for accessibility". Possibly a link to somewhere that goes into more detail about why this is the case.

Reply:
this is addressed later in the conversation @james-alt

Reply:
removed at @james-alt request